app/services/email_reply_service.rb

```
@@ -0,0 +1,366 @@
+
+          # Fall back to most recent
+          record = QuestionRecord.most_recent_for(group.id, question_id).first
+          return record if record
+        end
+      end
+    end
+
+    # If we get here, all identification methods failed
+    @identification_method = "failed"
+    nil
+  end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for identify_question_record is too high. [<24, 39, 22> 50.8/23]

(Metrics/AbcSize)